### DISCUSSION TITLE
for webGL

### DISCUSSION THREAD

**Discussion by tigran95ifix** [Voltstro-Studios/UnityWebBrowser#277] [General]
Hello. I came across your solution and it's amazing. It works very stably on Windows. I intend to use your solution in WebGL in **3d canvas** and for web pages that have "**X-Frame-Options limitation**". Please tell me, is this real?
Best regards

**Comment by Voltstro** [comment]
I don't understand your question. Are you saying you want to run UWB in a WebGl player? If so, then that is not possible.

**Comment by tigran95ifix** [comment]
ok, thanks
